plugins/modules/route53_health_check.py

```diff
@@ -419,31 +483,39 @@ def main():
         disabled=dict(type='bool'),
         ip_address=dict(),
         port=dict(type='int'),
-        type=dict(required=True, choices=['HTTP', 'HTTPS', 'HTTP_STR_MATCH', 'HTTPS_STR_MATCH', 'TCP']),
+        type=dict(choices=['HTTP', 'HTTPS', 'HTTP_STR_MATCH', 'HTTPS_STR_MATCH', 'TCP']),
         resource_path=dict(),
         fqdn=dict(),
         string_match=dict(),
         request_interval=dict(type='int', choices=[10, 30], default=30),
         failure_threshold=dict(type='int', choices=[1, 2, 3, 4, 5, 6, 7, 8, 9, 10]),
         tags=dict(type='dict', aliases=['resource_tags']),
         purge_tags=dict(type='bool'),
+        health_check_id=dict(type='str', aliases=['id'], required=False),
+        health_check_name=dict(type='str', aliases=['name'], required=False),
+        use_unique_names=dict(type='bool', required=False),
     )
 
     args_one_of = [
-        ['ip_address', 'fqdn'],
+        ['ip_address', 'fqdn', 'health_check_id'],
     ]
 
     args_if = [
         ['type', 'TCP', ('port',)],
     ]
 
+    args_required_together = [
+        ['use_unique_names', 'health_check_name'],
```

Comment:
Just asking: health_check_id and health_check_name shouldn’t be mutually exclusive?